js/repl/styles.js

```diff
@@ -12,14 +13,35 @@ const colors = {
   inverseBackground: "#21252b",
   inverseBackgroundDark: "#181a1f",
   inverseBackgroundLight: "#292d36",
-  inverseForeground: "#ffffff",
+  inverseForeground: "#faffff",
```

Comment:
The `inverseForeground` is modified from `#fff` to be `lab(100, 0, -5)` so that it shares the same colour channel with the `inverseBackground` which is `lab(15, 0, -5)`.